lib/company/opening.js

```diff
@@ -165,8 +165,55 @@ function createNewOpening(userDetails, openingDetail, callback) {
     }
   );
 }
+/**
+ * Function to update job opening of company
+ * @param {ObjectId} companyId company Id
+ * @param {Object} openingData having job opening Data
+ * @param {Object} callback has two parameters first errorInSave and Saved opening Details
+ */
+
+function updateJobOpening(companyId, openingData, callback) {
```

Comment:
you have `userId` saved in the session, you need to get the companyId by using the helper function for `getCompanyDetailsFromUser` from userId
Then you can implement this function.